### DISCUSSION TITLE
Is is possible to test the developer version?

### DISCUSSION THREAD

**Discussion by locksoft** [diasurgical/DevilutionX#7027] [Q&A] [answered]
Sorry for the (maybe) noob question, but I'm really a noob with Github. :)
Is it possible to download the developer version binary? I've read people reporting bugs in the 1.6 developer version but I've no idea if and where I can download it and/or if it's available for all the platforms. If I click on "releases" I can only see the official ones.

Hmmm... or maybe it means I just have to download the source code and compile it myself (operation that is way way out of my league)? :)

Sorry and thanks.

**Comment by MaderatorCreator** [comment, marked as answer]
https://github.com/diasurgical/devilutionX

Scroll down to the “Test builds” item and select the one you need.

**Reply by locksoft** [reply]
Oh, thank you. :)

**Comment by MaderatorCreator** [comment]
Keep in mind that your DevX data in test builds may be distorted or completely deleted. I would recommend installing test builds separately from the main version if possible. For example, I downloaded a test version on Android and, using Lucky Patcher, cheated the signature verification in the apk file so that the build would be installed separately.

**Reply by locksoft** [reply]
I see, thanks. I guess I can place the file in a different folder, and both games should appear on switch (hopefully).
